### PR TITLE
david's raw Bio

### PR DESCRIPTION
**Pull request by alan-cugler**
Davids Biography to add to our team page.

### CODE REVIEW

**Review by BenjaminEchelmeier — COMMENTED** [review]
Looks like you made changes to the theme?

File: content/who-we-are/our-team/contents.lr [hunk 1]
```diff
@@ -109,6 +109,27 @@ caption:
 background_color: 
 -----
 padding: 
+#### subnav-item ####
+body:
+
+G'day, I'm David Antonini.
+
+I've always had a thirst for knowledge and love of stories, delighting in reading about the whys and origins of things. At university I undertook majors from physics through to performing arts.
```
Comment (alan-cugler) [inline content/who-we-are/our-team/contents.lr:117]: flesh this out more.

Comment (alan-cugler) [reply content/who-we-are/our-team/contents.lr:117]: whats the connection between stories, origins, and physics.

File: content/who-we-are/our-team/contents.lr [hunk 2]
```diff
@@ -109,6 +109,27 @@ caption:
...
+
+My earliest memories of computers involve cassettes, stacks of floppy disks, and MS DOS prompts. I've long been enamored of the possibilities of technology, and get enthused by possibilities it holds for simple and elegant solutions to complex problems. After a career in the healthcare and education fields, I taught myself python and Linux, and now I look forward to learning more technologies and using them to provide the solutions to those problems.
```
Comment (alan-cugler) [inline content/who-we-are/our-team/contents.lr:119]: tech as a hobby is cool but the python/linux line is sales pitchy

**Review by alan-cugler — COMMENTED** [review]
I would like more here about you, you've lived in a few places, experienced lots of life. lets see some of that. what about family, values, and aspirations?

File: content/who-we-are/our-team/contents.lr (hunk 1 above)
Comment (toonarmycaptain) [reply content/who-we-are/our-team/contents.lr:117]: Well, originally someone told a child a story about where fire comes from, and now we have physics.

**Comment by alan-cugler** [comment]
@nixjdm I think this is good to Merge

**Pull request merged**